core/controllers/profile.py

```diff
@@ -148,6 +148,43 @@ def post(self, _):
         self.render_json({})
 
 
+class SubscribeToAndroidList(base.BaseHandler):
```

Comment:
Can you name this as a noun, like the other controllers in this file? Maybe AndroidListSubscriptionHandler or similar.

Reply:
Done.